invenio_rest/views.py

```diff
@@ -192,16 +203,16 @@ def dispatch_request(self, *args, **kwargs):
         """Dispatch current request.
 
         Dispatch the current request using
-        :py:meth:`flask.views.MethodView.dispatch_request` then, if the result
-        is not already a :py:class:`Flask.Response`, search for the
+        :class:`flask.views.MethodView` `dispatch_request()` then, if the
```

Comment:
it returns `WARNING: py:meth reference target not found: flask.views.MethodView.dispatch_request`
it looks like they didn't document it "directly".